ArcGISRuntimeSDKQt_QMLSamples/Maps/ChangeBasemap/ChangeBasemap.qml

```diff
@@ -40,74 +37,47 @@ Rectangle {
         anchors {
             left: parent.left
             top: parent.top
-            margins: 15 * scaleFactor
+            margins: 5
+        }
+        property int modelWidth: 0
+        width: modelWidth + leftPadding + rightPadding + indicator.width
+        textRole: "text"
+        model: ListModel {
+            ListElement { text: "Topographic";                  map: "BasemapTopographic" }
```

Comment:
personally, I wouldn't tab align these. We don't do it elsewhere and it is a maintenance headache if we change the list of names (e.g. a longer name gets inserted) . Either just separate with a space or put the map value on a 2nd line